src/interface/src/app/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="forget-password-container">
+  <div class="forget-password-card">
+    <h1 class="title">Reset Password</h1>
+
+    <p class="reset-text">Type in a new password for your account.</p>
+
+    <form [formGroup]="form" (ngSubmit)="submit()" class="email-form">
+      <mat-form-field appearance="fill">
+        <mat-label>Create password</mat-label>
+        <input type="password" required formControlName="password1" matInput />
+        <mat-error *ngIf="form.get('password1')?.hasError('minlength')">
+          Password must contain at least 8 characters.
+        </mat-error>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Confirm password</mat-label>
+        <input type="password" required formControlName="password2" matInput />
+        <mat-error *ngIf="form.hasError('passwordsNotEqual')">
+          Passwords must match.
+        </mat-error>
+      </mat-form-field>
+
+      <div class="reset-form-buttons">
+        <button
```

Comment:
I believe the button needs to have a type="button", is there a type cancel?

Reply:
Thanks. Didn't know that type can only be button|reset|submit. Changed to button.